### PR TITLE
fix(i18n): add urdu localization

### PR DESCRIPTION
**Pull request by raisedadead**
Recreates #273

### CODE REVIEW

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo if you don't mind can you add the review comments here again? And BTW https://github.com/freeCodeCamp/news/runs/6125626233?check_suite_focus=true#step:6:33, some same issues as before.

**Comment by ghost** [comment]
I have made the changes locally, but I do not have the necessary rights to push to `feat/urdu`.

image

**Comment by raisedadead** [comment]
> I have made the changes locally, but I do not have the necessary rights to push to `feat/urdu`.

Yes - that is because you keep flipping between multiple accounts. 😅 Please log in and use @zairahira, and delete the @zairaFcc account.

A fresh invite can be accepted from the @zairahira account here: https://github.com/freeCodeCamp

File: i18n/locales/urdu/translations.json [hunk 1]
```diff
@@ -1,80 +1,81 @@
...
+    "label": "سرچ کریں",
+    "accessible-name": "اپنی search query لکھیں ",
+    "placeholder": "متعدد tutorials میں سے search کریں ",
+    "see-results": "  کے سارے results دیکھیں {{ searchQuery }}",
+    "no-tutorials": "کوئی ٹٹوریل نہیں مل سکا "
   },
   "banner": {
     "default": "{{ <0> }}کوڈ کرنا سیکھیں 3000 گھنٹوں کا مفت نصاب{{ </0> }}",
     "authenticated": "ہمارے non profit mission کی حمایت کریں. {{ <0> }}freecodecamp کو عطیہ دیں{{ </0> }}",
     "authenticated-donor": "اپنے {{ </0> }}عطیات{{ <0> }} کے ذریعے freecodecamp کی حمایت کرنے کا شکریہ."
```
Comment (scissorsneedfoodtoo) [inline i18n/locales/urdu/translations.json]:
```suggestion
    "authenticated": "ہمارے non profit mission کی حمایت کریں. {{ <0> }}freeCodeCamp کو عطیہ دیں{{ </0> }}",
    "authenticated-donor": "اپنے {{ </0> }}عطیات{{ <0> }} کے ذریعے freeCodeCamp کی حمایت کرنے کا شکریہ."
```

File: i18n/locales/urdu/translations.json [hunk 2]
```diff
@@ -1,80 +1,81 @@
...
   "social-row": {
-    "author-has-twitter": "If you read this far, tweet to the author to show them you care. {{ <0> }}Tweet a thanks{{ </0> }}",
-    "author-no-twitter": "If this article was helpful, {{ <0> }}tweet it{{ </0> }}.",
-    "default-tweet": "Thank you {{ twitter }} for writing this helpful article."
+    "author-has-twitter": "اگر آپ نیں آخر تک پڑھ لیا ہے، تو مصنف کو شکریہ کریں. {{ <0> }} شکریہ کا Tweet کریں {{ </0> }}",
+    "author-no-twitter": " {{ <0> }}Tweet  کریں{{ </0> }} اگر یہ آرٹیکل مفید لگا تو ",
+    "default-tweet": "آپ کا بہت شکریہ، آپ نیں یہ آرٹیکل لکھا.  {{ twitter }} ."
   },
-  "learn-to-code-cta": "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers. {{ <0> }}Get started{{ </0> }}",
+  "learn-to-code-cta": "مفت میں coding کرنا سیکھیں. freecodecamp کے open source نصاب نے 40،000 سے زیادہ لوگوں کو ڈویلپر جاب ملنے میں مدد کی ہے. {{ <0> }} شروع کریں{{ </0> }}",
```
Comment (scissorsneedfoodtoo) [inline i18n/locales/urdu/translations.json]:
```suggestion
  "learn-to-code-cta": "مفت میں coding کرنا سیکھیں. freeCodeCamp کے open source نصاب نے 40،000 سے زیادہ لوگوں کو ڈویلپر جاب ملنے میں مدد کی ہے. {{ <0> }} شروع کریں{{ </0> }}",
```

File: i18n/locales/urdu/translations.json [hunk 3]
```diff
@@ -1,80 +1,81 @@
...
-    "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
-    "donation-initiatives": "Donations to freeCodeCamp go toward our education initiatives, and help pay for servers, services, and staff.",
-    "donate-text": "You can {{ <0> }}make a tax-deductible donation here{{ </0> }}.",
+    "tax-exempt-status": "freeCodeCamp ایک ڈونر سپورٹڈ ٹیکس سے مستثنیٰ 501(c)(3) غیر منفعتی تنظیم ہے (امریکہ کا وفاقی ٹیکس شناختی نمبر: 82-0779546)",
+    "mission-statement": "ہمارا مشن: لوگوں کو مفت میں کوڈ سیکھنے میں مدد کرنا۔ ہم اسے ہزاروں ویڈیوز، مضامین، اور انٹرایکٹو کوڈنگ اسباق بنا کر پورا کرتے ہیں - یہ سب عوام کے لیے  publically دستیاب ہیں۔ ہمارے پاس دنیا بھر میں ہزاروں FreeCodeCamp اسٹڈی گروپس بھی ہیں۔",
+    "donation-initiatives": "FreeCodeCamp کو عطیات ہمارے تعلیمی اقدامات کی طرف جاتے ہیں، اور سرورز، services اور عملے کی تنخواہ کی ادائیگی میں مدد کرتے ہیں۔",
```
Comment (scissorsneedfoodtoo) [inline i18n/locales/urdu/translations.json]:
```suggestion
    "mission-statement": "ہمارا مشن: لوگوں کو مفت میں کوڈ سیکھنے میں مدد کرنا۔ ہم اسے ہزاروں ویڈیوز، مضامین، اور انٹرایکٹو کوڈنگ اسباق بنا کر پورا کرتے ہیں - یہ سب عوام کے لیے  publicly دستیاب ہیں۔ ہمارے پاس دنیا بھر میں ہزاروں freeCodeCamp اسٹڈی گروپس بھی ہیں۔",
    "donation-initiatives": "freeCodeCamp کو عطیات ہمارے تعلیمی اقدامات کی طرف جاتے ہیں، اور سرورز، services اور عملے کی تنخواہ کی ادائیگی میں مدد کرتے ہیں۔",
```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Sorry @zairahira / @zairaFcc, I missed what could be a few minor typos in my last review.

Could you take a look and apply these suggestions if you agree?

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks for your patience @zairahira.

Gave this another quick look locally and everything LGTM :+1: :+1:

**Pull request merged**